rai/node/rpc.cpp

```diff
@@ -4124,6 +4152,7 @@ void rai::rpc_connection::parse_connection ()
 					boost::property_tree::write_json (ostream, tree_a);
 					ostream.flush ();
 					auto body (ostream.str ());
+					boost::replace_all (body, "\"__empty_array__\"", "[]");
```

Comment:
This is rather expensive on large output (since the entire buffer is scanned regardless of the presence of empty arrays)